providers/arista/resources/arista.lr.manifest.yaml

```diff
@@ -17,7 +17,7 @@ resources:
     min_mondoo_version: 5.15.0
     platform:
       name:
-      - arista-eos
+      - Arista EOS
```

Comment:
Those names match the platform name identifier. They should stay the way they in that document. If we need better names in the docs, we need to adjust the docs command to make this happen.

Reply:
Ah, thank you. Well is there any reason why we should reveal the platform name identifier to the readers of the docs?